### PR TITLE
Add base docs

### PR DESCRIPTION
**Pull request by evrardjp**

### CODE REVIEW

**Comment by jsuchome** [comment]
So the README content was _not_ changed in this PR, right?

**Comment by evrardjp** [comment]
Nope, just converted from md to rst using pandoc.

File: doc/test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+#
```
Comment (Itxaka) [inline doc/test-requirements.txt]: can we drop this file?

Comment (evrardjp) [reply doc/test-requirements.txt]: I guess I used what is the convention, but we can ofc remove it in a later patchset.

**Pull request merged**